students/wachulec_maria/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,13 @@
+number = int(input("Number: "))
+large_number = number
+sum_of_large_number = 1
+while True:
+    number = int(input("Number: "))
+    if number > large_number:
+        large_number = number
+        sum_of_large_number = 1
+    elif number == large_number:
+        sum_of_large_number += 1
+    if number == 0:
+        print(sum_of_large_number)
+        break
```

Comment:
it would be better not to use `break` and put a condition to while loop .e.g: `while n != 0:`